android-sdk/android/src/main/kotlin/io/logto/sdk/android/auth/logto/LogtoAuthManager.kt

```diff
@@ -0,0 +1,36 @@
+package io.logto.sdk.android.auth.logto
+
+import android.annotation.SuppressLint
+import android.net.Uri
+
+internal object LogtoAuthManager {
+    private var logtoAuthCallbackUriScheme: String? = null
+    @SuppressLint("StaticFieldLeak")
```

Comment:
what's this? should we add some comments for it?

Reply:
The linter finds that this is a global object, so it has a warning that this field may be leaked.
I think it's nothing to comment on here.